testdata/p4_16_samples_outputs/issue2147-1-midend.p4

```diff
@@ -20,16 +17,9 @@ control ingress(inout bit<8> h) {
         }
         const default_action = a();
     }
-    @hidden table tbl_issue21471l7 {
-        actions = {
-            issue21471l7();
-        }
-        const default_action = issue21471l7();
-    }
     apply {
         tbl_issue21471l5.apply();
         tbl_a.apply();
-        tbl_issue21471l7.apply();
```

Comment:
Here is a similar situation. Even though the body of the action a was empty, the compiler added two assignments:

    b = tmp;
    tmp = b;

The writes set of that action had both tmp and b, even though the second assignment was removed (after propagation), so the propagation after the action call wasn't possible.